kalite/version.py

```diff
@@ -5,10 +5,15 @@
 # THIS IS USED BY settings.py.  NEVER import settings.py here; hard-codes only!
 # Must be PEP 440 compliant: https://www.python.org/dev/peps/pep-0440/
 # Must also be of the form N.N.N for internal use, where N is a non-negative integer
-MAJOR_VERSION = "0"
-MINOR_VERSION = "14"
-PATCH_VERSION = "0"
-VERSION = "%s.%s.%s" % (MAJOR_VERSION, MINOR_VERSION, PATCH_VERSION)
+
+from django.utils.version import get_version
+
+VERSION_TUPLE = (0, 14, 0, "alpha", 0)
+
+MAJOR_VERSION, MINOR_VERSION, PATCH_VERSION = map(str, VERSION_TUPLE[:3])
+
+VERSION = get_version(VERSION_TUPLE)
```

Comment:
All I'm saying is that I'm not in a position to evaluate whether this is ok if, e.g. `VERSION_TUPLE = (0, 14, 0, "alpha", 42)`. I get that this is already the case on the alpha branches. In my mind that might be an issue, just because I was told the version string had to have a certain format, though I don't know why.

Also `get_version((0, 14, 0, "final" ,0))` does in fact return something different from before. `0.14` vs `0.14.0`. I would just like someone else to clarify what is required internally for our version numbers.